cluster/saltbase/salt/kube-addons/init.sls

```diff
@@ -0,0 +1,79 @@
+{% if pillar.get('enable_cluster_monitoring', '').lower() == 'true' %}
```

Comment:
YAML has an explicit bool type that turns into a python bool via salt..  It is a little odd that we are doing a string compare here instead.